WTG.Analyzers/Analyzers/BooleanLiteral/ArgumentListExtensions.cs

```diff
@@ -26,7 +26,19 @@ static class ArgumentListExtensions
 				return null;
 			}
 
-			var argumentSymbol = methodSymbol.Parameters[index];
+			var parameterSymbols = methodSymbol.Parameters;
+
+			if (index >= parameterSymbols.Length)
+			{
+				var lastParameter = parameterSymbols[parameterSymbols.Length - 1];
+
+				if (lastParameter.IsParams)
```

Comment:
Is it possible to reach this point if the last parameter is not params? If we can't, then this check is probably redundant. If we can then we will probably still get the IndexOutOfRangeException below.

Reply:
I don't think so - at least, not as of C# 9.0 - but if it is possible or becomes possible, then I would rather find out from a crash rather than silently ignore it.